rc/extra/autorestore.kak

```diff
@@ -44,19 +48,20 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
 ## Remove all the backups that have been created for the current buffer
 def autorestore-purge-backups -docstring "Remove all the backups of the current buffer" %{
-    nop %sh{
+    %sh{
         if [ ! -f "${kak_buffile}" ]; then
             exit
         fi
 
         buffer_basename="${kak_bufname##*/}"
         buffer_dirname=$(dirname "${kak_bufname}")
+        log=$(rm -v "${buffer_dirname}/.${buffer_basename}.kak."* 2>&1)
```

Comment:
`rm -v` isn't [POSIX](http://pubs.opengroup.org/onlinepubs/009695399/utilities/rm.html), and you can use `-f` to avoid error messages.

Reply:
Sorry about that. Old habits. I'm still trying to get used to posix.